site/en/docs/extensions/whatsnew/index.md

```diff
@@ -16,6 +16,12 @@ tags:
 Check this page often to learn about changes to the Chrome extensions platform, its documentation,
 and related policy or other changes. You'll find other notices posted on the [Extensions Google Group](https://groups.google.com/a/chromium.org/g/chromium-extensions). The [Extensions News](/tags/extensions-news/) tag lists articles about many of the topics listed here. (It even has [an RSS feed](/feeds/extensions-news.xml).)
 
+### Chrome 112: Increased storage.session quota {: #m112-storage-session-quota}
+
+March 1, 2023
+
+From Chrome 112, the quota for the [`storage.session`](/docs/extensions/reference/storage/#property-session) API has been increased to approximately 10 MB. This was discussed in the Web Extensions Community Group: [https://github.com/w3c/webextensions/issues/350](https://github.com/w3c/webextensions/issues/350)
```

Comment:
```suggestion
From Chrome 112, the quota for the [`storage.session`](/docs/extensions/reference/storage/#property-session) property has been increased to approximately 10 MB. This was discussed in the Web Extensions Community Group: [https://github.com/w3c/webextensions/issues/350](https://github.com/w3c/webextensions/issues/350)
```

Reply:
This is a best practice to use definite, specific, concrete language.

Reply:
>This was discussed in the Web Extensions Community Group

Was there a resolution to the discussion? I see that other vendors are adding to storage, but I can't find a PR to update the spec. The future we're headed toward is what happens on the rest of the DevRel team: everything is discussed in the standards committee so we only call it out when there's a resolution or some reason to draw attention to a specific discussion.

Reply:
Yes, increasing the limit to 10 MB was the resolution in this case. Unfortunately there isn't a spec for WebExtensions - the goal of the community group in the short term is just to align as best as possible. I wanted to call this out specifically because it's a nice example of us using the group to guide our direction.

Reply:
Then I think the statement needs to be more positive, something along the lines of 'vendors agreed to increase storage.' (I'm just trying to give you s sense. Word smith it your way.) One word of caution: if any of the vendors who were present at the meeting didn't agree, don't use phrasing that implies that they did.

Reply:
That makes sense. Thanks for the advice and approval!